tests/test_size_validators/test_are_repos_to_large.py

```diff
@@ -0,0 +1,34 @@
+import os.path
+from fiasko_bro.pre_validation_checks import are_repos_too_large
+
+
+def test_repo_size_fail_single():
+    max_py_files_count = 1
+    path_to_repo = 'test_fixtures{}general_repo'.format(os.path.sep)
```

Comment:
The `path_to_repo` and `path_to_original_repo` are fixtures and are duplicated across different tests. In order to avoid the copy-paste, do something like this: https://github.com/devmanorg/fiasko_bro/blob/master/tests/test_general_validators/conftest.py#L9 (except return the string instead of the repo). 

For additional info regarding fixtures, see https://docs.pytest.org/en/latest/fixture.html#fixtures-as-function-arguments